docs/_automation/logicApp/waste-reduction-logic-app.md

```diff
@@ -0,0 +1,166 @@
+# Automated Detection
```

Comment:
Don't forget to add the page metadata like other doc pages.

Reply:
Will add this to a new PR